rpcs3/Emu/Cell/Modules/cellVdec.cpp

```diff
@@ -492,16 +484,15 @@ s32 cellVdecClose(ppu_thread& ppu, u32 handle)
 		return CELL_VDEC_ERROR_ARG;
 	}
 
-	lv2_obj::sleep(ppu);
-	vdec->out_max = 0;
+	vdec->out_max.raw() = 0;
```

Comment:
Misuse of raw() method. And in general, this commit is unjustified. Creating interrupt threads in this manner gives more accurate control over thread execution via cmd_list in current implementation, generally speaking. Yes, it needs a hack in interrupt disestablish syscall. And so what, it's perfectly placed, and it can be even hidden against undesired execution with some flag so it won't even be possible to write a test which can prove its inaccuracy.

Reply:
Is there a reason the liblv2 function is used instead of the actual syscall? otherwise I can just remove the thread with idm directly and thats it.

Reply:
Its causing a race on clearing vdec thread's TLS, details on discird.